application_manager/plugins/sahara/plugin.py

```diff
@@ -532,7 +562,7 @@ def to_dict(self):
         }
 
     def execute(self, data):
-        app_id = "osspark" + self.id_generator.get_ID()
+        app_id = str(uuid.uuid4())[0:7]
```

Comment:
Is there a reason to use only the first eight characters as the "app_id"?

Reply:
No. Just seems to me a reasonable number for an ID.

Reply:
ok, it's fine for me.